buildSrc/src/main/java/org/elasticsearch/gradle/test/rest/RestResourcesExtension.java

```diff
@@ -32,6 +32,7 @@
 
     final RestResourcesSpec restApi;
     final RestResourcesSpec restTests;
+    private String sourceSetName = "yamlRestTest";
```

Comment:
Do we have to define this here _and_ in each of the task implementations? Can we have this default just defined on place?

Reply:
I ended up removing this entirely from the rest resource extension. The which source set to use is now decided by if the YamlRestTestPlugin has been applied. [`6289b00` (#56841)](https://github.com/elastic/elasticsearch/pull/56841/commits/6289b00b7fb71856968e2a6fa543202f4aa8cf7d)